packages/business-logic-plugin/BusinessLogicBase.ts

```diff
@@ -23,25 +23,25 @@ export class BusinessLogicBase implements BusinessLogicPlugin {
     eventFilter: object | null = null;
 
     startTransaction(req: Request, businessLogicID: string, tradeID: string): void {
-        // NOTE: This method implements the BisinessLogcPlugin operation(* Override by subclass)
+        // NOTE: This method implements the BusinessLogcPlugin operation(* Override by subclass)
```

Comment:
@jscode017 Is this meant to be `BusinessLogicPlugin`?

Reply:
Yeah sorry for that